### DISCUSSION TITLE
Make the date picker in `more info` localized

### DISCUSSION THREAD

**Discussion by oncleben31** [home-assistant/frontend#11275] [Other feature requests]
## The request

In the `more info` for datetime inputs, make the date picker localized with language and format consistent with user settings.

## The alternatives

None identified.

## Additional information

It seems the date picker used in the front end manage localization: https://vaadin.com/components/vaadin-date-picker/java-examples/localization

**Comment by mycanaletto** [comment]
+1

**Comment by clemalex824** [comment]
+1

**Comment by frenchyoda** [comment]
Still not available ?

**Comment by diestrohs** [comment]
What do you think about a pop-up spinner for date and time? [https://github.com/home-assistant/frontend/discussions/24326](url)
